### PR TITLE
fix #77,#80,#65

### LINKED ISSUE #77: [DOCS] Contributing Guidelines index is outdated
The index at the beginning of https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.rst is outdated.
This doc portion should be updated: 

```
Before submitting your contribution, please make sure to take a moment and read through the following guidelines :

[Code of Conduct](https://github.com/ltelab/disdrodb/blob/main/CODE_OF_CONDUCT.md)
[Issue Reporting Guidelines](https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.rst#issue-reporting-guidelines)
[Pull Request Guidelines](https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.rst#pull-request-guidelines)
[Development Setup](https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.rst#development-setup)
[Project Structure](https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.rst#project-structure)
[Github Flow](https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.rst#github-flow)
[Commit Lint](https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.rst#commit-lint)
```

### PR DESCRIPTION
**Pull request by regislon**
# Prework

- [x] I understand and agree to this repository's [code of conduct](https://github.com/ltelab/disdrodb/blob/main/CODE_OF_CONDUCT.md).
- [x] I understand and agree to this repository's [contributing guidelines](https://github.com/ltelab/disdrodb/blob/main/CONTRIBUTING.md).
- [x] I have already submitted an [issue](https://github.com/ltelab/disdrodb/issues) or [discussion thread](https://github.com/ltelab/disdrodb/discussions) to discuss my idea with the maintainers.

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and communicate accordingly:

**The PR fulfills these requirements:**

- [x] It's submitted to the branch named as follow : 
  - Add a reader: `reader-<institute>-<campaign>`
  - Fix a bug: `bugfix-<some_key>-<word>`
  - Improve the doc: `doc-<some_key>-<word>`
  - Add a new feature: `feature-<some_key>-<word>`
  - Refactor some code: `refactor-<some_key>-<word>`
  - Optimize some code: `optimize-<some_key>-<word>`
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] Don't forget to link PR to issue if you are solving one.
- [x] All tests are passing.
- [x] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

# Related GitHub issues and pull requests

- Ref: #77,#80,#65

# Summary
Update doc and remove all warings (errors in the code)

### CODE REVIEW

**Comment by regislon** [comment]
Hi @charlottegiseleweil , I've changed the install instructions as we discussed this morning. Could you check please ?

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -7,18 +7,21 @@ You can contribute in many ways :
...
+- Report `issues <#issue-reporting-guidelines>`__
+- Help us developping new readers 
+- Any others code improvements are welcome !
+
+
+Before adding your contribution, please make sure to take a moment
+and read through the following documnents :
+
+- `Code of Conduct <https://github.com/ltelab/disdrodb/blob/main/CODE_OF_CONDUCT.md>`__
+- `Contrubuting environement setup <#contributing-environment-setup>`__
```
Comment (ghiggi) [inline CONTRIBUTING.rst:19]:
```suggestion
- `Contributing environment setup <#contributing-environment-setup>`__
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -7,18 +7,21 @@ You can contribute in many ways :
...
+- `Contrubuting process <#contributing-process>`__
```
Comment (ghiggi) [inline CONTRIBUTING.rst:20]:
```suggestion
- `Contributing process <#contributing-process>`__
```

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -27,7 +30,8 @@ Issue Reporting Guidelines
    templates <https://github.com/ltelab/disdrodb/issues/new/choose>`__
 -  If you don’t find the required GitHub issue template, please ask for a new template.
 
-Contribution Guidelines
+
+Github
```
Comment (ghiggi) [inline CONTRIBUTING.rst:34]:
```suggestion
GitHub
```

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -216,7 +164,69 @@ Please define the name of your branch based on the scope of the contribution. Tr
...
 
-Here again, respect the `Code Style guide`_.
+
+We follow the pep8 and the python-guide writing style
+
+-  `Code Style — The Hitchhiker's Guide to
+   Python <https://docs.python-guide.org/writing/style/>`__
+
+To ensure a minimal style consistency, we use
+`black <https://black.readthedocs.io/en/stable/>`__ to auto-format to
```
Comment (ghiggi) [inline CONTRIBUTING.rst:174]:
```suggestion
`black <https://black.readthedocs.io/en/stable/>`__ to auto-format
```

File: disdrodb/L0/check_configs.py [hunk 5]
```diff
@@ -6,7 +6,7 @@
 @author: ghiggi
 """
 import numpy as np
-from disdrodb.standards import (
+from disdrodb.L0.standards import (
```
Comment (ghiggi) [inline disdrodb/L0/check_configs.py:9]: Nice catch !!!

File: docs/source/api/disdrodb.L0.readers.rst [hunk 6]
```diff
@@ -1,6 +1,17 @@
 disdrodb.L0.readers package
 ===========================
 
+Submodules
+----------
+
+disdrodb.L0.readers.parser\_template module
+-------------------------------------------
+
+.. automodule:: disdrodb.L0.readers.parser_template
```
Comment (ghiggi) [inline docs/source/api/disdrodb.L0.readers.rst:10]:
```suggestion
.. automodule:: disdrodb.L0.readers.reader_template

```

File: docs/source/install.rst [hunk 7]
```diff
@@ -65,102 +58,114 @@ According to the `contributors guidelines <contributors_guidelines>`__, you shou
...
-		conda env create -f environment.yml 
+		* Mac/Linux: Create a virtual environment with venv:
+
+			.. code-block:: bash
+
+			   virtualenv -p python3 venv
+			   source venv/bin/activate
+			   pip install -r requirements.txt
+
+	**With Conda env :**
```
Comment (ghiggi) [inline docs/source/install.rst:99]:
```suggestion
	**With Conda:**
```

File: docs/source/install.rst [hunk 8]
```diff
@@ -65,102 +58,114 @@ According to the `contributors guidelines <contributors_guidelines>`__, you shou
...
+			.. warning::
+				Note: In future, when the conda disdrodb feedstock installation will be set up, the following code should be used instead: 
+			
+				.. code-block:: bash
+			
+					conda install --only-deps disdrodb
+			
+		* Install disdrodb 
+		
+			To install the project in editable mode :
```
Comment (ghiggi) [inline docs/source/install.rst:128]: This section should not be indented under conda env only. 
This applies to whatever environment (venv and conda). 

pip install -e is the way to install the package in dev/editable mode.
The alternative is to set the PYTHONPATH described here below.

File: docs/source/install.rst [hunk 9]
```diff
@@ -65,102 +58,114 @@ According to the `contributors guidelines <contributors_guidelines>`__, you shou
...
-			   python -m venv venv
-			   cd venv/Script
-			   activate
+	* Create the `disdrodb-dev` conda environment and install the required dependencies:
 
-		* Mac/Linux: Create a virtual environment with venv:
+		.. code-block:: bash
 
-			.. code-block:: bash
+			conda env create -f environment.yml
```
Comment (ghiggi) [inline docs/source/install.rst:147]: This is the alternative to using `pip install requirements.txt` 
I would move it above, just before or above the note on `conda install --only-deps disdrodb`

File: docs/source/readers.rst [hunk 10]
```diff
@@ -55,43 +49,43 @@ There are a couple of optional parameters that can added to the previous command
 	* The path should end with <campaign_name> and match the end of raw_dir.
 	* Example: '<...>/disdrodb/data/processed/<campaign_name>'.
 
-* ``--l0a_processing`` : bool [ **true** |false] - Whether to launch processing to generate L0A Apache Parquet file(s) from raw data.
+* ``--l0a_processing`` : bool [ **true** \|false] - Whether to launch processing to generate L0A Apache Parquet file(s) from raw data.
```
Comment (ghiggi) [inline docs/source/readers.rst:52]:
```suggestion
* ``--l0a_processing`` : bool [ **true** \|false] - Whether to launch processing to generate DISDRODB L0A Apache Parquet file(s) from raw data.

```

File: docs/source/readers.rst [hunk 11]
```diff
@@ -55,43 +49,43 @@ There are a couple of optional parameters that can added to the previous command
 	* The path should end with <campaign_name> and match the end of raw_dir.
 	* Example: '<...>/disdrodb/data/processed/<campaign_name>'.
 
-* ``--l0a_processing`` : bool [ **true** |false] - Whether to launch processing to generate L0A Apache Parquet file(s) from raw data.
+* ``--l0a_processing`` : bool [ **true** \|false] - Whether to launch processing to generate L0A Apache Parquet file(s) from raw data.
 
 
 
-* ``--l0b_processing`` : bool [ **true** |false] - Whether to launch processing to generate L0B netCDF4 file(s) from L0A data.
+* ``--l0b_processing`` : bool [ **true** \|false] - Whether to launch processing to generate L0B netCDF4 file(s) from L0A data.
```
Comment (ghiggi) [inline docs/source/readers.rst:56]:
```suggestion
* ``--l0b_processing`` : bool [ **true** \|false] - Whether to launch processing to generate DISDRODB L0B netCDF4 file(s) from L0A data.

```

File: docs/source/readers.rst [hunk 12]
```diff
@@ -55,43 +49,43 @@ There are a couple of optional parameters that can added to the previous command
...
-* ``--lazy`` : bool [ **true** |false] - Whether to perform processing lazily with dask.
+* ``--lazy`` : bool [ **true** \|false] - Whether to perform processing lazily with dask.
 
 	* If lazy=True, it employed dask.array and dask.dataframe.
 	* If lazy=False, it employed pandas.DataFrame and numpy.array.
 
 
 
-* ``--single_netcdf`` : bool [ **true** |false] - Whether to concatenate all raw files into a single L0B netCDF file.
+* ``--single_netcdf`` : bool  [ **true** \| false] - Whether to concatenate all raw files into a single L0B netCDF file.
```
Comment (ghiggi) [inline docs/source/readers.rst:88]:
```suggestion
* ``--single_netcdf`` : bool  [ **true** \| false] - Whether to concatenate all raw files into a single DISDRODB L0B netCDF file.

```

**Review by ghiggi — COMMENTED** [review]
Nice ! 
I added some suggestions. 
I think the section on the installation in dev mode still needs some polish (see the comments)

File: docs/source/install.rst (hunk 8 above)
Comment (regislon) [reply docs/source/install.rst:128]: @ghiggi Let's keep that for the next PR if you dont mind

File: docs/source/install.rst (hunk 9 above)
Comment (regislon) [reply docs/source/install.rst:147]: @ghiggi We had a discussion on that with @charlottegiseleweil. If you don't mind, I close thisPR and we discuss that on monday morning together.

**Pull request merged**